frontend/src/static/sideNav.json

```diff
@@ -13,6 +13,12 @@
                 "icon": "virtual-column--key",
                 "link": "/grant",
                 "disabled": true
+            },
+            {
+                "name": "Add application admin",
+                "icon": "enterprise",
+                "link": "#",
+                "disabled": true
             }
         ]
     }
```

Comment:
This `sideNav.json` is a bit strange, isn't it similar to the `routeItem.ts`?
If they are different purpose, at least why we code the:
*  "**link**" with value duplicated and same as "**path**" in `routeItems`?
* And "**name**" with value duplicated and same as "**label**" in `routeItems`?

Reply:
this is to auto populate the side nav, you just add a new here and the nav will display

Reply:
Maybe you didn't understand what I meant.
Probably better to unify them, especially, if the **path**s or **link**s can be out of sync.
Maybe you and @NickSaglioni can think about it later, I will ignore it for now.
I will just post screenshot of what I saw.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/7af139fc-d273-49c4-9d22-c2807d66d4d9)